src/read_sample_config.py

```diff
@@ -38,10 +38,16 @@ def read_sample_config(config_f):
 
             samples_dict[sample] = {"vcf": vcf, "bam": bam}
 
+            # expect only filepath per field
             for colname in ["capture_bed"]:
                 if colname in row:
                     samples_dict[sample][colname] = is_valid_file(row[colname])
```

Comment:
```suggestion
            if "capture_bed" in row:
                samples_dict[sample][colname] = is_valid_file(row[colname])
```
I missed this before but there's only 1 column being looked at here so the use of a loop isn't needed and the column name can just be checked for individually in the if block

Reply:
never mind on this, I saw in #90 that this is setup for a larger structure that just isn't realized in the PR